src/exercise/06.extra-1.js

```diff
@@ -0,0 +1,29 @@
+// Basic Forms
```

Comment:
I don't think we need this file because the finished version of 06 is enough to continue on to the extra credit. We only create an extra credit exercise file when the starting point is different from the ending point of the exercise.